### DISCUSSION TITLE
第22期：多邻国真的可以帮助学英语吗？ | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#45] [General]
# 第22期：多邻国真的可以帮助学英语吗？ | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-10-21_022

**Comment by WymanY** [comment]
> 你可以拿我的这个idea去设计开发一款自己的应用，只要把学习过程设计得更有趣味性，能够满足用户情绪价值，或许能出现一个比多邻国更有效的产品。

这里已经有一个开源的软件：https://github.com/ZuodaoTech/everyone-can-use-english，跟你的想法很像。另外我使用多邻国学习英语，并不是为了提高英语，只是单纯的使用一个工具提高语感。而且多邻国会不厌其烦的提醒你今天没学，所以它额外起到一个监督者的作用，真正的提高英语肯定是通过系统的学习。

**Reply by WymanY** [reply]
这就是大概我明知道它作用不大，依然在使用它的原因。人都是要监督的。如果你这个周刊一直没有读者，相信你能坚持的动力会小一点，这就是自我监督和他监督的区别

**Comment by pojuy673** [comment]
点赞~

**Comment by xiaobinji** [comment]
很棒的分享。谢谢
